src/Dropdown/Dropdown.svelte

```diff
@@ -123,12 +125,12 @@
     highlightedIndex = index;
   }
 
-  $: if (selectedIndex > -1) {
+  $: if (selectedId !== undefined) {
     dispatch("select", { selectedId, selectedIndex, selectedItem });
```

Comment:
I'm not sure if we should dispatch `selectedIndex` 🤔

Reply:
Let's leave it in for now

Reply:
Done in d52828a 🙆‍♂️